src/main/java/ru/yandex/practicum/filmorate/controllers/UserController.java

```diff
@@ -43,22 +42,22 @@ public User updateUser(@RequestBody @Valid User user) {
     }
 
```

Comment:
Если сделать такой пост http://localhost:8080/users
{
	"name2":"nameItem",
	"email":"email3@email.com",
  "login":"login"
}
То на выходе получается exception
{
	"error": "Unknown",
	"description": "Cannot invoke \"java.time.LocalDate.getYear()\" because \"date\" is null"
}

java.lang.NullPointerException: Cannot invoke "java.time.LocalDate.getYear()" because "date" is null
	at java.sql/java.sql.Date.valueOf(Date.java:291) ~[java.sql:na]
	at ru.yandex.practicum.filmorate.dao.impl.UserDaoImpl.lambda$add$0(UserDaoImpl.java:36)